src/components/Header/index.tsx

```diff
@@ -287,6 +287,10 @@ const StyledChevron = styled(ChevronUp)<{ isOpen: boolean }>`
   transform: ${({ isOpen }) => (isOpen ? 'rotate(0deg)' : 'rotate(180deg)')};
 `
 
+export const SpacedSkeleton = styled(Skeleton)`
+  margin-right: 3px;
+`
+
```

Comment:
I would suggest to move it inline styles like this style={{ marginRight: '3px' }} since its just used in 2 components. Or at least to place it is some different component if its being exported like this

Reply:
Moved the style to inline margin